rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -51,13 +51,13 @@ PIfElse.      Proc1 ::= "if" "(" Proc ")" Proc2 "else" Proc1 ;
 PNew.         Proc1 ::= "new" [NameDecl] "in" Proc1 ;
 PPar.         Proc  ::= Proc "|" Proc1 ;
 
-separator nonempty Proc "," ;
+separator Proc "," ;
 
 -- Names
 NameWildcard. Name ::= "_" ;
 NameVar.      Name ::= Var ;
 NameQuote.    Name ::= "@" Proc12 ;
-separator nonempty Name "," ;
+separator Name "," ;
```

Comment:
Same as above

Reply:
This enables empty lists, and empty sends/receives. I told him to do it.